lib/Perl/Metrics/Lite/Analysis/File.pm

```diff
@@ -51,7 +51,14 @@ sub _init {
         = Perl::Metrics::Lite::Analysis::Util::get_packages($document);
 
     my @sub_analysis = ();
-    my $sub_elements = $document->find('PPI::Statement::Sub');
+    #my $sub_elements = $document->find('PPI::Statement::Sub');
```

Comment:
there is one file which generates uninitialised access so maybe we should dot it this way - unless you come with the better option

         return '' unless $_[1]->parent == $_[0] 
            and 
            ($_[1]->isa('PPI::Statement::Sub') 
            or ($_[1]->isa('PPI::Statement') 
            and ( defined $_[1]{children}[0]{content} and $_[1]{children}[0]{content} eq 'rpc'))
            );

that was the error - Use of uninitialized value in string eq at /Users/leonidsokolovskiy/projects/p5-perl-metrics-lite/blib/lib/Perl/Metrics/Lite/Analysis/File.pm line 59.

Reply:
Applied. Seems everything is ok now.